crates/iota-rest-api/src/transactions/execution.rs

```diff
@@ -68,7 +68,7 @@ impl ApiEndpoint<RestService> for ExecuteTransaction {
 async fn execute_transaction(
     State(state): State<Option<Arc<dyn TransactionExecutor>>>,
     Query(parameters): Query<ExecuteTransactionQueryParameters>,
-    client_address: Option<axum::extract::ConnectInfo<SocketAddr>>,
+    client_address: Result<axum::extract::ConnectInfo<SocketAddr>, ExtensionRejection>,
```

Comment:
why did we change that one?

Reply:
Its an axum change, it does no longer take an Option, but instead it takes a Result now. Took me some time to figure out why, ended up in https://github.com/tokio-rs/axum/discussions/3245#discussioncomment-12389573